app/reminder/views.py

```diff
@@ -1,29 +1,29 @@
 import datetime
 
-from flask import (
-    Blueprint,
-    abort,
-    flash,
-    redirect,
-    render_template,
-    request,
-    url_for,
-)
+from flask import (Blueprint, abort, flash, redirect, render_template, request,
+                   url_for)
+from flask_login import login_required
+
+from app.decorators import *
 
-from .forms import NewReminderForm
 from .. import db
 from ..models import Reminder
+from .forms import NewReminderForm
 
 reminder = Blueprint('reminder', __name__)
 
 
 @reminder.route('/', methods=['GET'])
+@login_required
 def dashboard():
     """Dashboard to view and add text and email reminders."""
-    return render_template('reminder/index.html', reminders=Reminder.query.all())
+    return render_template(
+        'reminder/index.html', reminders=Reminder.query.all())
 
 
 @reminder.route('/new', methods=['GET', 'POST'])
+@login_required
+@admin_required
```

Comment:
I think this should also include `@advisor_required`

Reply:
or i'm not sure if there will be a specific route for user specific reminders? @katiejiang

Reply:
No user specific reminders

Reply:
Don't think we need to worry too much about that